Description: Increase the task stack to work around stack corruption
Author: Karol Ossowski <karol.ossowski@gmail.com>
Bug-Debian: https://bugs.debian.org/863039

--- wvstreams-4.6.1.orig/utils/wvtask.cc
+++ wvstreams-4.6.1/utils/wvtask.cc
@@ -429,7 +429,7 @@ void WvTaskMan::_stackmaster()
 	    total = (val+1) * (size_t)1024;
 	    
             if (!use_shared_stack())
-                total = 1024; // enough to save the do_task stack frame
+                total = 2048; // enough to save the do_task stack frame
 
 	    // set up a stack frame for the new task.  This runs once
 	    // per get_stack.
